docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -2747,7 +2747,7 @@ scope :chronological, -> { order(created_at: :asc) }
 
 | Enabled
 | Yes
-| No
+| Yes
```

Comment:
I think this shouldn't be considered a safe autocorrect. For example, depending on the Rails log level configuration, changing from `puts` to `Rails.logger.debug` could result in no output being shown.

Reply:
That's true! Thanks!